share/completions/git.fish

```diff
@@ -549,7 +549,10 @@ function __fish_git_needs_command
 end
 
 function __fish_git_config_keys
-    git config -l | string match -r '[^=]+'
+    # Print already defined config values first
+    git config -l | string replace '=' \t
```

Comment:
Unfortunately config values can be multiline, so this would have to use `git config -lz`:

```fish
git config -lz | while read -lz key value
    printf '%s\t%s\n' $key (printf %s $value)[1]
end
```

- The "-z" format uses a space, not an "=" as a separator, so reading into two variables just works
- The inner "printf" is to just use the first line of each value

Reply:
Ah right, I made the changes and also appended an ellipsis to the first line of multiline values.